docs/frontend_utilisation_guide.md

```diff
@@ -24,48 +24,92 @@ The Pixel Streaming Frontend UI contains all the functionality for UI components
+npm npm install
+
+# Symlink the library to the example
+npm run symlink
+
+# Build the example
+npm run build-dev
 ```
+These commands and scripts will install the example, link the library to the example and build the example.
+
+#### Installing and building with the build-all script
+1) Download the [Scalable Pixel Streaming Frontend source code from GitHub](https://github.com/ScalablePixelStreaming/Frontend)
 
-### Building the Scalable Pixel Streaming Frontend
-1) Run the following command in the base directory of the source tree:
+2) Run the following script in the `examples/typescript` directory of the source tree:
 ```bash
-# Install the frontend library's dependencies
-npm install .
+# Install, link and build the example and library
+npm run build-all-dev
 ```
-This will transpile the TypeScript source files to JavaScript and generate the UMD module in the dist subdirectory. 
+This will install, link and build the Scalable Pixel Streaming Frontend example and library all in one.
 
-### Implementing the Scalable Pixel Streaming Frontend in a custom webpage
-The Scalable Pixel Streaming Frontend comes with a base index.ts file which is the entry point. In a custom webpage this file can be left out and a custom entry point can be implemented: 
+### Installing the Scalable Pixel Streaming Frontend through NPM
+If your project includes a `package.json` file run the following command in the same directory
+1) Run the following command: `npm i @tensorworks/libspsfrontend`
 
-1) Import all the required packages from the Epic Games Pixel Streaming Frontend and Frontend UI libraries
-```typescript
-import { Config, PixelStreaming } from '@epicgames-ps/lib-pixelstreamingfrontend-ue5.2';
-import { PixelStreamingApplicationStyle } from '@epicgames-ps/lib-pixelstreamingfrontend-ui-ue5.2';
-```
-2) Import SPSApplication from the Scalable Pixel Streaming Frontend Library 
+2) Import your desired components form the library from the following package `"@tensorworks/libspsfrontend"` 
+
+#### Basics to initialising and consuming the library
+The following example for initialising the library is based on the TypeScript example Provided on Github however is more broad. 
+
+1) Import all the required packages from the Scalable Pixel Streaming Frontend library
 ```typescript
-import { SPSApplication } from "./SPSApplication";
+import {Config, PixelStreaming, SPSApplication, TextParameters, PixelStreamingApplicationStyle} from "@tensorworks/libspsfrontend";
 ```
-3) Create `PixelStreamingApplicationStyles` variable so we have access to the Epic Games Pixel Streaming CSS: 
+2) Apply default styling from Epic Games Pixel Streaming Frontend 
 ```typescript
 export const PixelStreamingApplicationStyles = new PixelStreamingApplicationStyle();
+PixelStreamingApplicationStyles.applyStyleSheet();
 ``` 
+3) Create a `webSocketAddress` variable so the websocket url can be modified if a user wishes to inject their own websocket address at load time
```

Comment:
should WebSocket URL be capitalised here?

Reply:
Resolved in 4820831c542f48909f428f142606b7c814b6ede6